front/src/applications/operationalStudies/v2/components/ScenarioHeader.tsx

```diff
@@ -33,7 +48,7 @@ const ScenarioHeader = ({ scenario, infra }: ScenarioHeaderProps) => {
   const scenarioNameRef = useRef<HTMLSpanElement>(null);
   const usernameRef = useRef<HTMLButtonElement>(null);
 
-  const boards = ['Traits', 'Map', 'Macro', 'STD', 'SDD', 'Table', 'Conflicts'];
+  const boards = ['Trains', 'Map', 'Macro', 'STD', 'SDD', 'Table', 'Conflicts'];
```

Comment:
@clarani noticed this morning that we're missing translations for these.

Reply:
The translations are given in the main [ticket](https://github.com/osrd-project/osrd-confidential/issues/1063)
| Anglais | Français | Allemand |
|----|----|----|
| Trains | Trains | **TODO** |
| overview | Vue d'ensemble | **TODO** |
| STDs | GETs | **TODO** |
| TODs | GOVs | **TODO** |
| SDDs | GEVs | **TODO** |
| Tables | Tableaux | **TODO** |
| Conflicts | Conflits | **TODO** |